src/shared/util/graphql.util.ts

```diff
@@ -0,0 +1,28 @@
+import { Logger } from '@nestjs/common'
+import { GraphQLClient, gql } from 'graphql-request'
+
+const ballotReadyKey = process.env.BALLOT_READY_KEY || ''
+
+export async function queryHelper(query: any, variables?: any) {
+  const logger = new Logger('queryHelper')
+  try {
+    const endpoint = 'https://bpi.civicengine.com/graphql'
+
+    const graphQLClient = new GraphQLClient(endpoint, {
+      headers: {
+        authorization: `Bearer ${ballotReadyKey}`,
+        'Content-Type': 'application/json',
+      },
+    })
+
+    const gqlQuery = gql`
+      ${query}
+    `
+
+    const data: any = await graphQLClient.request(gqlQuery, variables || {})
+    return data
+  } catch (e) {
+    logger.error('error at queryHelper', e)
+    throw e
+  }
+}
```

Comment:
Pretty sure this should be a service w/ the `GraphQLClient` as a `private` singleton on the service class, instead of instantiating a new instance w/ each and every request.

The `endpoint` can either be passed into the service, if we're going to be using this as an interface to talk to multiple different GraphQL services, or it can just be a standalone constant in this file instead of creating a new string literal for each request as well.